lib/runners/processor/phpmd.rb

```diff
@@ -46,50 +48,48 @@ def target_files
 
     def rule
       rules = config_linter[:rule] || config_linter.dig(:options, :rule)
-      if rules
-        rules
-      else
-        # NOTE: It assumes that default config xml is located in $HOME
-        (Pathname(Dir.home) / 'sider_config.xml').realpath.to_s
-      end
+      comma_separated_list(rules) || DEFAULT_CONFIG_FILE
     end
 
     def target_dirs
-      Array(config_linter[:target] || './').flat_map { |target| target.split(',') }.join(',')
+      comma_separated_list(config_linter[:target] || DEFAULT_TARGET)
```

Comment:
Should `target_dirs` always return a String value? If so, how about this?

```suggestion
      comma_separated_list(config_linter[:target]) || DEFAULT_TARGET
```

Reply:
Thanks. Your suggestion is right! 👍